Library/Styles/BaseStyles.swift

```diff
@@ -119,5 +125,12 @@ private let baseNavigationBarStyle =
     NSForegroundColorAttributeName: UIColor.black,
     NSFontAttributeName: UIFont.ksr_callout()
     ]
-    <> UINavigationBar.lens.translucent .~ false
```

Comment:
ah sorry, i thought the default was `false` and it seemed to work, but now i see it's wrong. fixing...